config/suit/templates/nrf54h20/default/v1/root_with_binary_nordic_top.yaml.jinja2

```diff
@@ -185,77 +217,180 @@ SUIT_Envelope_Tagged:
-      - suit-send-record-failure
-    # In the case of streaming operations it is worth to retry them at least once.
-    # This increases the robustness against bit flips on the transport,
-    # for example when storing the data on an external memory device.
-    # The suit-directive-try-each will complete on the first successful subsequence.
-    - suit-directive-try-each:
-      - - suit-condition-image-match:
+      # Radio payload is unavailable. Proceed with application verification.
+      - - suit-directive-set-component-index: 0
+        - suit-directive-override-parameters:
+            suit-parameter-uri: '#{{ application['name'] }}'
+            suit-parameter-image-digest:
+              suit-digest-algorithm-id: cose-alg-sha-256
+              suit-digest-bytes:
+                envelope: {{ artifacts_folder ~ application['name'] }}.suit
+        - suit-directive-fetch:
+          - suit-send-record-failure
+        - suit-condition-image-match:
           - suit-send-record-success
           - suit-send-record-failure
           - suit-send-sysinfo-success
           - suit-send-sysinfo-failure
-      - - suit-condition-image-match:
+    {%- if radio is defined %}
+        # Ensure that the application manifest version matches exactly the installed radio manifest version.
+        - suit-directive-set-component-index: {{ rad_component_index }}
+        - suit-directive-override-parameters:
+            suit-parameter-version:
+              suit-condition-version-comparison-equal: {{ APP_LOCAL_1_VERSION }}
+            suit-parameter-soft-failure: False
+        - suit-condition-version:
           - suit-send-record-success
           - suit-send-record-failure
           - suit-send-sysinfo-success
           - suit-send-sysinfo-failure
-    - suit-condition-dependency-integrity:
-      - suit-send-record-success
-      - suit-send-record-failure
-      - suit-send-sysinfo-success
-      - suit-send-sysinfo-failure
-    - suit-directive-process-dependency:
-      - suit-send-record-success
-      - suit-send-record-failure
-      - suit-send-sysinfo-success
-      - suit-send-sysinfo-failure
+    {%- endif %}
 {%- endif %}
+
+{%- if radio is defined %}
+    - suit-directive-run-sequence:
+      - suit-directive-set-component-index: 0
+      - suit-directive-override-parameters:
+          suit-parameter-uri: '#{{ radio['name'] }}'
+          suit-parameter-image-digest:
+            suit-digest-algorithm-id: cose-alg-sha-256
+            suit-digest-bytes:
+              envelope: {{ artifacts_folder ~ radio['name'] }}.suit
+          suit-parameter-soft-failure: True
+      - suit-directive-fetch:
+        - suit-send-record-failure
+      - suit-condition-image-match:
+        - suit-send-record-success
+        - suit-send-record-failure
+        - suit-send-sysinfo-success
+        - suit-send-sysinfo-failure
+      - suit-directive-override-parameters:
+          suit-parameter-soft-failure: False
```

Comment:
This part really requires to add a comment about the assumptions and why it is safe to run `dependency-integrity` and `process-dependency` here, instead of the `try-each` validating update scenarios.

Reply:
Please take a look now.